### PR TITLE
Update CHANGELOG and migrations for 1.19.4

### PR DESCRIPTION
**Pull request by gthao313**
**Issue number:**

**Description of changes:**
Updates CHANGELOG and migrations for 1.19.4 release

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# v1.19.4 (2024-04-06)
+
+## OS Changes
+* Update kernel to 5.10.213, 5.15.152, 6.1.82 ([#3865])
+* Update Containerd ([#3869])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* update containerd to 1.6.31 ([#3869])
```

File: Release.toml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version = "1.20.0"
+version = "1.19.4"
```
Comment (bcressey) [inline Release.toml]: The version here should stay at 1.20.0.

File: Twoliter.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 schema-version = 1
-release-version = "1.20.0"
+release-version = "1.19.4"
```
Comment (bcressey) [inline Twoliter.toml]: Ditto - keep this at 1.20.0.

**Review by bcressey — COMMENTED** [review]
The pull request title and the description should be improved.

**Review by jmt-lab — APPROVED** [review]

**Comment by gthao313** [comment]
Base on offline talk with ben, we decide to keep 1.20.0 on twolitter and release files, and we still bump to 1.19.4 on 1.19.x branch.

**Review by jmt-lab — APPROVED** [review]

**Review by jpculp — APPROVED** [review]

**Review by rpkelly — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**